package.json

```diff
@@ -33,10 +33,10 @@
     "d3": "3.5.17",
     "tangram.cartodb": "cartodb/tangram.cartodb#master",
     "jquery": "2.1.4",
-    "leaflet": "0.7.3",
+    "leaflet": "1.0.3",
     "mustache": "1.1.0",
     "perfect-scrollbar": "git://github.com/CartoDB/perfect-scrollbar.git#master",
-    "torque.js": "CartoDB/torque#master",
+    "torque.js": "CartoDB/torque#leaflet-migration",
```

Comment:
Yeah! We've to release a version with this PR https://github.com/CartoDB/torque/pull/284, but not until this it's merged.